jar-service/src/main/java/com/sixsq/slipstream/authn/CookieAuthenticator.java

```diff
@@ -98,10 +98,9 @@ private boolean handleValid(Request request, Cookie cookie) {
 
 		logger.fine("handle valid, cookie = " + cookie);
 		String tokenInCookie = CookieUtils.tokenInCookie(cookie);
-		if(tokenInCookie!=null) {
+		if(tokenInCookie != null) {
 			logger.fine("handle valid, tokenInCookie = " + tokenInCookie);
-			user.setAuthnToken(tokenInCookie);
-			user = user.store();
+			user.storeAuthnToken(tokenInCookie);
```

Comment:
Find a way to conditionally store the authn token during the Cookie based authentication. https://github.com/slipstream/SlipStreamServer/issues/582